react-native-purchases-ui/package.json

```diff
@@ -52,17 +52,17 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "devDependencies": {
-    "@types/jest": "^28.1.2",
+    "@types/jest": "^29.5.12",
```

Comment:
It would be nice to be able to share these so they are not repeated... but I'm ok with it for now!

Reply:
yes, there's probably a way of doing that, we should research